tensorflow/tools/ci_build/install/install_pip.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
No need for this file once https://github.com/tensorflow/tensorflow/pull/1472 is in.